packages/formation-react/src/components/AlertBox/AlertBox.jsx

```diff
@@ -45,7 +45,7 @@ class AlertBox extends React.Component {
     const closeButton = this.props.onCloseAlert && (
       <button
         className="va-alert-close"
-        aria-label="Close notification"
+        aria-label={this.props.closeBtnAriaLabel || 'Close notification'}
```

Comment:
I would be curious what others think about setting a default value for the `closeBtnAriaLabel` prop instead of doing the conditional here.

Reply:
That would be more idiomatic.

Reply:
Would also show up in the component docs if we do it that way.

Reply:
Added default prop & aXe test.